scripts/setup-coinbase-address.js

```diff
@@ -26,9 +24,8 @@ module.exports = async node => {
   // allow for runtime configuration of which
   // address to use for coinbase transactions
   // TODO: come up with generalized way to pass args to runtime scripts
-  const COINBASE_WALLET_ID = process.env.BPANEL_COINBASE_WALLET_ID || 'primary';
-  const COINBASE_ACCOUNT_ID =
-    process.env.BPANEL_COINBASE_ACCOUNT_ID || 'default';
+  const COINBASE_WALLET_ID = config.str('coinbase-wallet-id', 'primary');
+  const COINBASE_ACCOUNT_ID = config.str('coinbase-account-id', 'default');
```

Comment:
Will this parse the value of the environment variable `COINBASE_ACCOUNT_ID`?

Reply:
Or does the environment variable need to be `BPANEL_COINBASE_ACCOUNT_ID`?

Reply:
the way bcfg works is that it uses the `module` name (passed as the first argument on initialization of the Client object) to identify configuration variables. So it looks in `~/.[MODULE_NAME]` for configs and it looks for `[MODULE_NAME]_` for env variables.